azkaban-exec-server/src/main/java/azkaban/execapp/AzkabanExecutorServer.java

```diff
@@ -162,6 +167,21 @@ public AzkabanExecutorServer(Props props) throws Exception {
     }
 
     logger.info("Azkaban Executor Server started on port " + portNumber);
+
+    if (props.getBoolean(AzkabanConstants.IS_METRICS_ENABLED, false)) {
```

Comment:
Duplicate logic between AzkabanExecutorServer and AzkabanWebServer. Should we use AzkabanServer to share the common code ?